src/Command/SendNotificationCommand.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\Notification;
+
+class SendNotificationCommand
+{
+    private $elements;
+
+    public function __construct($elements)
+    {
+        $this->elements = iterator_to_array($elements[0]);
```

Comment:
This code instantiates all the elements while only one notification channel is used on the `send()` method call. Imagine there are hundreds of Notification implementations registered and all they are instantiated on SendNotificationCommand instantiation